template.yaml

```diff
@@ -3213,6 +3215,7 @@ Resources:
     Type: AWS::Lambda::EventSourceMapping
     Condition: IpvExists
     Properties:
+      Enabled: !If [ SpotResponseEventMappingDisabled, false, true ]
```

Comment:
I know this is a bit ugly but it will only take a boolean, not a condition 🙃

Reply:
Is it possible to add it as another condition on the resource (where `IpvExists` is) or do we still want it to be created, just not enabled?